README.md

```diff
@@ -7,7 +7,7 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+SQL databases are relational databases whereas NoSQL databases are non-relational databases.  Typically an SQL database is used with tables whereas a NoSQL database uses  collections and documents based.  NoSQL provides more flexibility because there is less structure.  An example of a NoSQL would be MongoDB.
 ```
 
```

Comment:
NoSQL is distinct database architecture from MongoDB, but both are non-relational DBs.